From 4e4c9f023d84e37de412f1f484615edca6628549 Mon Sep 17 00:00:00 2001
From: Paulo Cesar Pereira de Andrade <pcpa@mandriva.com.br>
Date: Sun, 16 Mar 2008 23:33:57 +0000
Subject: xf86dga doesn't require Xt, Xmu and Xaw.

  Remove inclusion of Xt/Xmu/Xaw headers and update includes to match
missing libc and libX11 protypes.
---
diff --git a/configure.ac b/configure.ac
index c9113c4..ad6543c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -32,7 +32,7 @@ AC_PROG_CC
 AC_PROG_INSTALL
 
 # Checks for pkg-config packages
-PKG_CHECK_MODULES(XF86DGA, x11 xxf86dga xt xaw7 xmu)
+PKG_CHECK_MODULES(XF86DGA, x11 xxf86dga)
 AC_SUBST(XF86DGA_CFLAGS)
 AC_SUBST(XF86DGA_LIBS)
 
diff --git a/dga.c b/dga.c
index 7305ecd..eb1ec6e 100644
--- a/dga.c
+++ b/dga.c
@@ -1,26 +1,16 @@
 /* $XFree86: xc/programs/xf86dga/dga.c,v 3.19 2001/10/28 03:34:32 tsi Exp $ */
 
-#include <X11/Xos.h>
-#include <X11/Intrinsic.h>
-#include <X11/Shell.h>
-#include <X11/StringDefs.h>
-#include <X11/Xatom.h>
-#include <X11/Xaw/Form.h>
-#include <X11/Xaw/Scrollbar.h>
-#include <X11/Xaw/Label.h>
-#include <X11/Xaw/Command.h>
-#include <X11/Xaw/AsciiText.h>
-#include <X11/Xaw/Box.h>
-#include <X11/Xaw/Toggle.h>
-#include <X11/Xmu/StdSel.h>
-#include <X11/Xmd.h>
+#include <X11/Xlib.h>
+#include <X11/Xutil.h>
 #include <X11/extensions/xf86dga.h>
+#include <sys/time.h>
 #include <ctype.h>
 #include <errno.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include <signal.h>
-
+#include <unistd.h>
 
 
 #define MINMAJOR 0
--
cgit v0.8.2
